resource/pkg_factories.hpp.em

```diff
@@ -27,12 +27,12 @@ from ros1_bridge import camel_case_to_lower_case_underscore
 #include <ros1_bridge/factory.hpp>
```

Comment:
Nit:
 ```
#include <memory>
#include <string>
```

Reply:
Added in 8e0eef5d563ee8765fca4b9d0e895e58ec107795.